.travis.yml

```diff
@@ -1,27 +1,26 @@
 dist: trusty
-group: edge
+sudo: required
+
 language: php
+
 cache:
   directories:
     - $HOME/.composer/cache/files
 
-php:
-  - 5.6
-  - 7.0
-  - 7.1
-  - 7.2
-
-env:
-  - VARNISH_VERSION=5.0
-
 matrix:
   include:
     - php: 5.6
       env: SYMFONY_VERSION=2.7.* VARNISH_VERSION=3.0 COMPOSER_FLAGS="--prefer-lowest"
     - php: 7.0
-      env:
-        - VARNISH_VERSION=4.1
-        - DOCCHECK=true
+      env: VARNISH_VERSION=4.1 DOCCHECK=true
+    - php: 7.0
+      env: VARNISH_VERSION=4.1 VARNISH_MODULES_VERSION=0.9.1
+    - php: 7.0
+      env: VARNISH_VERSION=5.0
+    - php: 7.1
+      env: VARNISH_VERSION=5.1 VARNISH_MODULES_VERSION=0.12.1
+    - php: 7.2
+      env: VARNISH_VERSION=5.2
```

Comment:
lets keep this out of this PR and do it separately to keep the scope of the PR to x-key support